environments/dev/main.jsonnet

```diff
@@ -34,6 +34,35 @@ local dex = (import '../../components/dex.libsonnet') + {
 };
 
 local obs = (import '../base/observatorium.jsonnet') + {
+  tls_secret: {
+    apiVersion: 'v1',
+    kind: 'Secret',
+    metadata: {
+      name: 'observatorium-api-tls-certs',
+      namespace: obs.config.namespace,
+    },
+    data: {
+      'server.key': importstr '../../tmp/base64_certs/observatorium/server.key',
```

Comment:
Let's keep this simple and do the base64 encoding inside of jsonnet, that way we don't need the extra steps in the makefile

Reply:
Done